src/commands/deploy/genezio.ts

```diff
@@ -817,13 +848,32 @@ export function getCloudAdapter(provider: CloudProviderIdentifier): CloudAdapter
     }
 }
 
-export function getFunctionHandlerProvider(functionType: FunctionType): AwsFunctionHandlerProvider {
+export function getFunctionHandlerProvider(
```

Comment:
i think @radu1122 added a new provider/function type: `httpServer`. we should make sure to update this function accordingly